frontend/src/libs/components/icon/icon.tsx

```diff
@@ -0,0 +1,14 @@
+import { type ValueOf } from '#/libs/types/types.js';
+import { type IconNameToIcon } from '#libs/enums/enums.js';
+
+import { iconNameToPlainSvgMap } from './icon-map.js';
+
+type Properties = {
+  name: ValueOf<typeof IconNameToIcon>;
+};
+
+const Icon: React.FC<Properties> = ({ name }) => (
+  <>{iconNameToPlainSvgMap[name]}</>
```

Comment:
@Ekit18 that is the reason of that console error. The logic should be something like this:
```jsx
const SelectedIcon = iconNameToIcon[name];

return <SelectedIcon className={getValidClasses(styles.icon, className)} />;
```